src/commands/setAppCreateOptionsDefault.ts

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import { Uri, workspace, WorkspaceConfiguration } from 'vscode';
+import { IAppCreateOptions, javaUtils, LinuxRuntimes, WebsiteOS } from 'vscode-azureappservice';
+import { IActionContext } from 'vscode-azureextensionui';
+import { configurationSettings, extensionPrefix } from '../constants';
+
+export async function setAppCreateOptionsDefault(actionContext: IActionContext): Promise<IAppCreateOptions> {
+    const isJavaProject: boolean = await javaUtils.isJavaProject();
```

Comment:
As mentioned in the shared tools PR, I think you should delete `IAppCreateOptions`. Since you moved everything over here, you can just use the wizardContext directly.